telegraf/1.21/alpine/Dockerfile

```diff
@@ -23,8 +29,7 @@ RUN set -ex && \
     mv /usr/src/telegraf*/etc/telegraf/telegraf.conf /etc/telegraf/ && \
     mkdir /etc/telegraf/telegraf.d && \
     cp -a /usr/src/telegraf*/usr/bin/telegraf /usr/bin/ && \
-    gpgconf --kill all && \
```

Comment:
Same question: are we certain we are ok to drop this?

Reply:
So, `gpgconf --kill all` was added so that we could remove `${GNUPGHOME}`. Without it, there's a [race condition](https://stackoverflow.com/questions/45856100/remove-gnupghome-error-during-dockerfile-build) where `rm` enumerates the files it deletes, but `gpg-agent` has already removed them. The link suggests `pkill`, but not all containers base images have `pkill` so we using `gpgconf` instead.

This still isn't enough, occasionally `rm` will still fail.

I've decided that it's okay to keep `$GNUPGHOME`. It only contains public keys and it only adds about 30KiB to the image.